js/source/clustered_geojson_worker_source.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var supercluster = require('supercluster');
+var GeoJSONWorkerSource = require('./geojson_worker_base');
+
+var workerSource = Object.create(GeoJSONWorkerSource);
+workerSource.indexData = function (data, params, callback) {
```

Comment:
Would it be appropriate to call this method `clusterData`? I'm not completely sure what the word "index" means in this context.